dynamic_rest/routers.py

```diff
@@ -260,17 +262,11 @@ class UserSerializer(..):
             return routes
 
         serializer = viewset.serializer_class()
-        fields = getattr(serializer, 'get_all_fields', serializer.get_fields)()
+        fields = getattr(serializer, 'get_link_fields', lambda: [])()
 
         route_name = '{basename}-{methodnamehyphen}'
 
         for field_name, field in six.iteritems(fields):
-            # Only apply to DynamicRelationFields
-            # TODO(ryo): Maybe also apply in cases where fields are mapped
-            #            directly to serializers?
-            if not isinstance(field, DynamicRelationField):
```

Comment:
Not strictly related to this PR, but in refactoring `fields`, importing DynamicRelationField was causing circular dependencies, and realized using `serializer.get_link_fields()` solved that problem and also made things more consistent.